api/manifest/manifest.go

```diff
@@ -782,12 +782,16 @@ func runPrefix(prefix, executable string, args ...string) error {
 
 var randomAlphabet = []rune("abcdefghijklmnopqrstuvwxyz")
 
-func randomString(size int) string {
+func randomString(prefix string, size int) string {
 	b := make([]rune, size)
 	for i := range b {
-		b[i] = randomAlphabet[rand.Intn(len(randomAlphabet))]
+		idx, err := rand.Int(rand.Reader, big.NewInt(int64(len(randomAlphabet))))
```

Comment:
Let's leave the simpler version of the `randomAlphabet[]`selection in place. Did this version get copied from somewhere else?